sisl/supercell.py

```diff
@@ -676,9 +677,15 @@ def scale(self, scale):
 
         Parameters
         ----------
-        scale : ``float``
-           the scale factor for the new lattice vectors
+        scale : ``float`` or array-like of floats with shape (3,)
+           the scale factor for the new lattice vectors.
+           If it's an arrray of shape (3,), each lattice vector is scaled
+           by a different factor.
         """
+        if isinstance(scale, Iterable):
```

Comment:
alternatively, simply do `(self.cell.T * scale).T` this would work regardless of check